pom.xml

```
@@ -7,7 +7,7 @@
 
     <groupId>de.jsilbereisen</groupId>
     <artifactId>perfumator-java</artifactId>
-    <version>0.3.1</version>
+    <version>0.3.2</version>
```

```suggestion
    <version>0.4.0</version>
```
nitpick: If you want to follow [semver](https://semver.org/), this should be a minor version increase.